WebRoot/css/zm.css

```diff
@@ -1716,6 +1750,14 @@ INPUT[type="text"].search_input-expanded:focus {
 	line-height:normal;
 }
 
+.EditorDragBdr {
```

Comment:
so change it to some other name or merge both properties if applied to same element?